app/vmalert/utils/metrics.go

```diff
@@ -0,0 +1,54 @@
+package utils
+
+import "github.com/VictoriaMetrics/metrics"
+
+type namedMetric struct {
+	Name string
+}
+
+// Unregister removes the metric by name from default registry
+func (nm namedMetric) Unregister() {
```

Comment:
It seems, that Unregister method can be added to the metrics package for each object, it improves user experience @valyala

Reply:
Created a FR https://github.com/VictoriaMetrics/metrics/issues/32